src/pages/ProductDetail/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Star } from 'react-feather';
+import './StarRating.scss';
+
+const StarRating = () => {
+  const [rating, setRating] = useState(0);
+
+  return (
+    <div className="star-rating">
+      {[...Array(5)].map((star, index) => {
```

Comment:
```suggestion
      {[...Array(5)].map((_, index) => {
```
star 매개변수는 사용되지 않고 있는 것 같습니다.
해당 부분 사용하지 않는다라는 의미로 언더바를 입력해주세요